phpunit.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="Bootstrap.php" colors="true">
+<phpunit bootstrap="module/EcampCore/test/Bootstrap.php" colors="true">
```

Comment:
Ich glaube es ist richtig, dass phpunit.xml vom Module EcampCore hin zur obersten Ebene angehoben wurde. 

Gleichzeitig glaube ich, dass dann aber auch das File Bootstrap.php nicht in diesem Modul liegen sollte. Ich glaube, dass das Verschieben der Bootstrap.php zwei Probleme lösen würde.
- Location von coverage_clover (module/EcampCore/test/build/logs/clover.xml)
- Es lassen sich auch Tests für andere Module erfassen (es ist dann nicht so EcampCore spezifisch)

Idee:
Evtl braucht es analog zum public-Folder eine Art test-Folder. Dort würden sich dann Bootstrap.php (analog zu index.php) und die Test-Resultate / Coverage-Resultate befinden.